src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -501,6 +571,15 @@ internal abstract partial class BaseSimpleAssignmentExpression : AssignmentExpre
             base(OperationKind.SimpleAssignmentExpression, syntax, type, constantValue)
         {
         }
+        public override IEnumerable<IOperation> Children
```

Comment:
we could, but I think all having same pattern make it easy for us to spot problem. only class that has Accept will have Children as well since that is outer most class that contains all properties needed for specific IOperation